requirements-test.txt

```diff
@@ -1,3 +1 @@
-coveralls
-django-extensions>=2.0.7
```

Comment:
`coveralls` is installed by openwisp-utils[qa] 
`django-extensions` is installed by openwisp-users